frontend-react/src/pages/admin/AdminRevHistory.tsx

```diff
@@ -0,0 +1,273 @@
+                </h4>
+                <section className="margin-bottom-5">
+                    Select different versions from each list to compare.
+                </section>
+
+                <GridContainer
+                    className={"rs-revision-history rs-maxwidth-vw80"}
+                >
+                    <Grid row className={"rs-list-diffs-container"}>
+                        <Suspense fallback={<Spinner />}>
+                            <MainRevHistoryComponent
+                                org={org || ""}
+                                settingType={settingType || "organization"}
+                                leftSelectedListItem={leftSelectedListItem}
+                                rightSelectedListItem={rightSelectedListItem}
+                                onClickHandler={onClickHandler}
+                            />
+                        </Suspense>
+                    </Grid>
+                    <Grid row>
+                        <Grid col={"fill"}>
+                            <StaticCompare
+                                leftText={leftJson}
+                                rightText={rightJson}
+                            />
+                        </Grid>
+                    </Grid>
+                    <Grid row>
+                        <Grid col={"fill"}>
+                            <Grid row gap="md">
+                                <Grid data-testid={"meta-left-data"}>
+                                    Flags: <br />
+                                    {!leftItem ? null : (
+                                        <>
+                                            isDeleted:{" "}
+                                            {String(leftItem.isDeleted)}
+                                            <br /> isActive:{" "}
+                                            {String(leftItem.isActive)}
+                                        </>
+                                    )}
+                                </Grid>
+                            </Grid>
+                        </Grid>
+                        <Grid col={"fill"}>
+                            <Grid row gap="md">
+                                <Grid data-testid={"meta-right-data"}>
+                                    Flags: <br />
+                                    {!rightItem ? null : (
+                                        <>
+                                            isDeleted:{" "}
+                                            {String(rightItem.isDeleted)}
+                                            <br /> isActive:{" "}
+                                            {String(rightItem.isActive)}
+                                        </>
+                                    )}
+                                </Grid>
+                            </Grid>
+                        </Grid>
+                    </Grid>
+                </GridContainer>
+            </section>
+            <HipaaNotice />
```

Comment:
???

Reply:
`<HipaaNotice />` seems to be on every page. I'm not sure how to fix this comment

Reply:
interesting, never noticed that (as evidenced by the fact that any pages I've built don't have it). Wonder what the actual requirement is. If it really needs to be everywhere should we add this info to the footer? All of that is out of scope for now